README.md

```diff
@@ -200,238 +45,16 @@ jobs:
         # Prints: Secret: ***
 ```
 
-<details>
-<summary><b>Usage example with Service Accounts</b></summary>
+## 💙 Community & Support
 
-```yml
-on: push
-jobs:
-  hello-world:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
+- File an [issue](link/to/issues) for bugs and feature requests.
```

Comment:
Please update the link here `https://github.com/1Password/load-secrets-action/issues`

Reply:
Whops, that slipped when I updated the template. Nice catch! 👍🏻